src/controllers/organization.controller.js

```diff
@@ -324,6 +324,37 @@ export const resyncOrganization = async (req, res) => {
   }
 };
 
+export const addMetadata = async (req, res) => {
+  try {
+    await assertIfReadOnlyMode();
+    await assertWalletIsSynced();
+    await assertHomeOrgExists();
+
+    transaction = await sequelize.transaction();
+
+    await Organization.update(
```

Comment:
also, metadata needs to be written to the datalayer, then its automatically synce to the database.